aead/Cargo.toml

```diff
@@ -13,11 +13,10 @@ documentation = "https://docs.rs/aead"
 repository = "https://github.com/RustCrypto/traits"
 keywords = ["crypto", "encryption"]
 categories = ["cryptography", "no-std"]
-rust-version = "1.56"
+rust-version = "1.65"
 
 [dependencies]
-crypto-common = "0.1.4"
-generic-array = { version = "0.14", default-features = false }
+crypto-common = "=0.2.0-pre"
```

Comment:
I wonder if we should use `crypto-common = { version = "=0.2.0-pre", path = "../crypto-common" }` here.

Reply:
It's pulling in that version via the `[patch.crates-io]` directive in the toplevel `Cargo.toml`, which avoids repeating it in every crate